### PR TITLE
Realese Note for #15361: Implement volumetric fog support for both po…

### LINKED ISSUE #1702: Write release notes for PR #15361: Implement volumetric fog support for both point lights and spotlights
https://github.com/bevyengine/bevy/pull/15361 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `/home/alice/Documents/bevy-website/release-content/0.15/release-notes/15361_Implement_volumetric_fog_support_for_both_point_lights_and.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by Soulghost**
Fixes #1702

### CODE REVIEW

File: release-content/0.15/release-notes/15361_Implement_volumetric_fog_support_for_both_point_lights_and.md [hunk 1]
```diff
@@ -1,4 +1,47 @@
-<!-- Implement volumetric fog support for both point lights and spotlights -->
-<!-- https://github.com/bevyengine/bevy/pull/15361 -->
+Volumetric fog was introduced in Bevy 0.14, where initially, only directional lights could interact with it. In Bevy 0.15, interactions with point lights and spot lights are also supported.
```
Comment (alice-i-cecile) [inline release-content/0.15/release-notes/15361_Implement_volumetric_fog_support_for_both_point_lights_and.md:1]:
```suggestion
Volumetric fog was [introduced in Bevy 0.14](https://bevyengine.org/news/bevy-0-14/#volumetric-fog-and-volumetric-lighting-light-shafts-god-rays). Initially, only directional lights could interact with it. In Bevy 0.15, point lights and spot lights work with it too.
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Small edits, but more importantly I'd really like to have a screenshot for this :) This is one of our few shiny rendering features this release.

**Comment by Soulghost** [comment]
Hi @alice-i-cecile , I would like to ask for advice on storing the updated volumetric fog screenshot. Should I link to [the existing URL](https://github.com/bevyengine/bevy-website/blob/main/static/screenshots/3D%20Rendering/volumetric_fog.png) and wait for it to update automatically by the GitHub action, or should I place the image elsewhere?

**Comment by alice-i-cecile** [comment]
This should be stored in the repo: check out the 0.14 release notes for where and how this is done. We should also remove the instruction text from the image for showcase purposes.

**Comment by Soulghost** [comment]
I have checked the release notes in 0.14 that contain images, both of them use a image path like this:

```
![](filmic_color_grading.jpg)
```

However, the `filmic_color_grading.jpg` is actually located at `content/news/2024-07-04-bevy-0.14/filmic_color_grading.jpg`, not in the same directory as the markdown file. I haven't figure out how the markdown file links to the associated image.

Below is the screenshot without help text. Could you guide me on how to properly link it to the markdown file? Thank you!
volumetric_fog_with_three_lights

**Comment by mweatherley** [comment]
> Could you guide me on how to properly link it to the markdown file? Thank you!

Now that there is a stub for the actual release notes page, what you'll want to do is merge the `main` branch into your PR branch and put the image into the `content/news/<blah-blah-0.15>` folder. When referencing it in markdown, it will just be `my-image.png` or whatever, where that's the name that you gave it in the folder.

**Review by cart — APPROVED** [review]
I added the image and rebased. Nice work!

**Pull request merged**